src/app/routes/frontend.server.routes.ts

```diff
@@ -0,0 +1,52 @@
+import { celebrate, Joi, Segments } from 'celebrate'
```

Comment:
this file should be moved into `src/app/modules/frontend/frontend.routes.ts`

Reply:
shifted